### ISSUE TITLE
In case of pagination, only able to get first page of table in pdf.

### ISSUE THREAD

**Issue by rborale14** [IzakMarais/reporter#111]
Is there any way to get full table in pdf.

**Comment by IzakMarais** [comment]
No, I don't think this is supported via Grafana's Share direct rendered image API, which is what the reporter uses. 

If you want to research, you could play around with Grafana, and see if you can render the image you are interested in, via Panel->sharing->direct rendered image. See/research if you can get the table panel to render the subsequent pages, perhaps via some parameters.
